README.md

```diff
@@ -56,7 +57,7 @@ to `./runner.yml` and overriding relevant properties.
 To set up the project run:
 
 ```bash
-$ ./vendor/bin/run drupal:site-setup
+./vendor/bin/run drupal:site-setup
```

Comment:
This should be removed like in other repositories we are now updating.

Reply:
Fixed.